test/functional/index.js

```diff
@@ -111,15 +111,13 @@ describe('Autocomplete', () => {
       })
 
       describe('menuAttributes', () => {
-        it('renders with the extra attribute on the menu', () => {
-          render(<Autocomplete menuAttributes={{ 'aria-labelledby': 'autocomplete-default' }} id='autocomplete-default' />, scratch)
+        it('renders with extra attributes on the menu', () => {
+          render(<Autocomplete menuAttributes={{ 'data-test': 'test' }} id='autocomplete-default' />, scratch)
 
           const wrapperElement = scratch.getElementsByClassName('autocomplete__wrapper')[0]
           const dropdownElement = wrapperElement.getElementsByTagName('ul')[0]
-          const inputElement = wrapperElement.getElementsByTagName('input')[0]
 
-          expect(dropdownElement.getAttribute('aria-labelledby')).to.equal('autocomplete-default')
-          expect(inputElement.getAttribute('id')).to.equal('autocomplete-default')
+          expect(dropdownElement.getAttribute('data-test')).to.equal('test')
         })
 
         describe('attributes computed by the component', () => {
```

Comment:
**issue** I think we'd still want a test in here to check that it defaults the `aria-labelledby` to the `id` (and ideally one that it can be overriden, but that one's not super critical).

Reply:
I've added a test for both checking that the labelledby is the id by default and that it gets overriden by `menuAttributes`